protocol/messenger_backup_handler.go

```diff
@@ -136,6 +136,13 @@ func (m *Messenger) handleBackedUpProfile(message *protobuf.BackedUpProfile, bac
 		}
 	}
 
+	for _, s := range message.SocialLinkSettings {
+		err = m.handleSyncSocialLinkSetting(state, *s)
```

Comment:
@qfrank why don't you use `WakuBackedUpDataResponse` but adding these details to `ReceivedMessageState`?

Reply:
I mean if you backing it up as a part of `protobuf.BackedUpProfile`, why don't you handle it that way as well and keep the separation between handling data coming from backed up messages against data coming from syncing?

Reply:
actually, i was struggled with this before, i agree you now 😂 @saledjenic thank you

Reply:
fixed @saledjenic